### PR TITLE
Text formatting in bellapps.md(Fixes #1953)

### LINKED ISSUE #1953: Text formatting in Bell apps course instructions
### Problem
The instructions for working with Bell, Planet or GitHub are formatted in the same way by highlighting the possible buttons or clicks which looks so good and are easy to grasp. But this formatting was missed at "Take the course" instructions. The difference can be seen in the following screenshots.

### Steps to reproduce the problem
N/A

### Screenshots
_Missed formatting in:_

image

_Correct formatting in:_

image

### Proposed solution
The first one can be formatted as the second one which gives a uniform and better look and feel

### PR DESCRIPTION
**Pull request by BhargaviNadendla**
Fixes #1953

### Description

Step-4 BellApps tutorial was updated with some text formatting. Here I attach the screenshot of it.

image
 

### RawGit preview link

https://rawgit.com/BhargaviNadendla/BhargaviNadendla.github.io/FormattingText_BellApps/#!pages/vi/vi-bellapps.md

### CODE REVIEW

File: pages/vi/vi-bellapps.md [hunk 1]
```diff
@@ -65,9 +65,9 @@ We created a course for aspiring virtual interns so that we could test the softw
 Take the course, and the last question will ask you to specify any problems or improvements/suggestions that you have. Remember, people taking these courses in remote areas of the world will probably run into the same problems if you do not raise them now and let the team know. This is a very important task, and your help is much appreciated.
 
 **STEPS:**
-1. Click on Courses on the top navigation
-2. Find the course "Virtual Interns," click on Admission
-3. Click on Open
+1. Click on `Courses` on the top navigation
+2. Find the course "Virtual Interns," click on `Admission`
```
Comment (Sriharsha-Singam) [inline pages/vi/vi-bellapps.md]: Do you think "Virtual Interns" could instead be: `Virtual Interns` ?

File: pages/vi/vi-bellapps.md [hunk 2]
```diff
@@ -65,9 +65,9 @@ We created a course for aspiring virtual interns so that we could test the softw
...
+3. Click on `Open`
 4. Carefully finish 12 steps of the course
```
Comment (Sriharsha-Singam) [inline pages/vi/vi-bellapps.md]: I feel like *12 steps* should also be `12 steps`

Comment (BhargaviNadendla) [reply pages/vi/vi-bellapps.md]: @ Sriharsha-Singam I felt to highlight the clickable or changeable elements as it was done below of that description.

image

Comment (akshayuncc) [reply pages/vi/vi-bellapps.md]: @BhargaviNadendla Seems fair.

Comment (Sriharsha-Singam) [reply pages/vi/vi-bellapps.md]: ok

**Pull request merged**